conans/model/package_ref.py

```diff
@@ -0,0 +1,28 @@
+from conan.api.output import ConanOutput
+from conan.internal.model.package_ref import PkgReference as _PkgReference
+
+_msg = """
+*******************************************************************
+private '{}'
+detected in user code (custom commands, extensions, recipes, etc).
+Stop using it, use only public documented APIs.
+*******************************************************************
+"""
```

Comment:
Minor thing but I think we may add a please there

```suggestion
_msg = """
*******************************************************************
Private '{}' detected in user code (custom commands, extensions, recipes, etc).
Please stop using it and use only public, documented APIs.
*******************************************************************
"""
```

Reply:
I'll add the "Please", but not moving the ``{}`` in the same line, as the import is long as the line wraps and it is uglier